privcount/tally_server.py

```diff
@@ -324,6 +334,15 @@ def refresh_config(self):
             logging.warning("problem reading config file: missing required keys")
             log_error()
 
+    def get_secret_handshake_path(self):
+        '''
+        Return the path of the secret handshake key file, or None if the config
+        has not been loaded.
+        '''
+        # The secret handshake path should be loaded (or assigned a default)
+        # whenever the config is loaded
+        return self.config.get('secret_handshake')
```

Comment:
Is it possible that `self.config` might be `None` at this point? If so, this line would cause a crash.